client/src/sagas/watchFetchMovies.js

```diff
@@ -0,0 +1,42 @@
+import {put, takeEvery} from 'redux-saga/effects';
+
+import SEARCH_FILTER_TYPE from '../components/SearchFilter/searchFilterType'
+import SORT_TYPE from '../components/Sort/sortType';
+import {MOVIES_FETCH_REQUESTED, MOVIES_FETCH_SUCCEEDED, MOVIES_FETCH_FAILED} from '../actions/actionTypes';
+
+const getFilterPart = (filterType, filterValue) => {
+    if (filterValue==="") return "";
```

Comment:
_trivial_
I believe, that it would become cleaner if you make this as a default case for the `switch` statement.